cross_section/compute_cross_section.py

```diff
@@ -0,0 +1,57 @@
+import argparse 
+import ROOT
+import yaml
+import sys
+sys.path.append('Utils')
+from AnalysisUtils import get_n_events_from_zorro
+
+def main(config_file_name):
+    '''
+    Compute the cross section using the provided configuration file.
+
+    Parameters:
+    - config_file_name (str): The path to the configuration file.
+
+    Returns:
+    None
+    '''
+    
+    with open(config_file_name, 'r') as yml_config_file:
+        config = yaml.load(yml_config_file, yaml.FullLoader)
+
+    analysis_results_files = config['lumi']['analysis_results_files']
+    n_events = get_n_events_from_zorro(analysis_results_files, config['lumi']['zorro_folder'], config['lumi']['triggers_of_interest'], config['lumi']['h_collisions_path'])
+    int_lumi = n_events/config['lumi']['tvx_cross_section']
+
+    br_b_to_d = 2.51/1000
+    br_d_to_pikpi = 9.38/100
```

Comment:
Maybe this can become a configurable (sometimes they get updated in the PDG) together with the tvx cross section
```suggestion
    br_b_to_d = config['br']['b0_todminuspi']
    br_d_to_pikpi = config['br']['dplus_tokpipi']
```

Reply:
Yes, better to do that, thank you!